test/system/test_cvp_client.py

```diff
@@ -31,6 +31,13 @@
 #
 
 ''' System test for the CvpClient class
+
+    Requirements for CVP Node:
+    1) Just need one node for test with the following account:
```

Comment:
Can these be (optionally) supplied another way instead of being hard coded?  I'm thinking of something like `eapi.conf`.  A `.cvprc`, perhaps?  cvprac as a whole might benefit from that.  I could add that as an enhancement request instead of holding this up.